scipy/special/_lambertw.h

```diff
@@ -96,7 +96,7 @@ namespace scipy {
 		if (k == 0) {
 		    return z;
 		}
-		sf_error("lambertw", SF_ERROR_SINGULAR, NULL);
+		set_error("lambertw", SF_ERROR_SINGULAR, NULL);
```

Comment:
I'm curious how there is no error here. We are using `SF_ERROR_SINGULAR` without it being explicitly defined. My guess is that `sf_error.h` is being included before `_lambertw.h` in the NumPy wrapper file.

We'll need to expose the error enum that defines `SF_ERROR_SINGULAR`, `SF_ERROR_SLOW`, and so on. Probably it should just be in `error.h`.

Reply:
Oh, right. yeah, It's just working accidentally in the context of SciPy. I don't have a better solution than just copying it into `error.h` so will just do that.